test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -975,11 +977,6 @@ TEST(mongoRegisterContext_update, NotifyContextAvailability2)
           .Times(1);
   setNotifier(notifierMock);
 
-  TimerMock* timerMock = new TimerMock();
```

Comment:
Same here... Maybe the (newly added) `uInit()` call is doing this, but I prefer you to confirm.